config/locales/en.yml

```diff
@@ -427,9 +427,9 @@ en:
     disregarded_benefits_guidance:
       text: Guidance on disregarded benefits
       certificated:
-        link: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1045682/Lord_Chancellor_s_guidance_on.pdf#page=36"
+        link: "%{page_url}"
```

Comment:
I think all these `"%{page_url}"` translations are now redundant and these lines can be deleted, as you're not using them anywhere.